docs/settings/reporting-settings.asciidoc

```diff
@@ -203,26 +203,35 @@ The `file:` protocol is always denied, even if no network policy is configured.
 ==== CSV settings
 
 [[xpack-reporting-csv]] `xpack.reporting.csv.maxSizeBytes` {ess-icon}::
-The maximum {byte-units}[byte size] of a CSV file before being truncated. This setting exists to prevent large exports from causing performance and storage issues. Can be specified as number of bytes. Defaults to `10mb`.
+The maximum {byte-units}[byte size] of a CSV file before being truncated. This setting exists to prevent large exports from causing performance and storage issues. Can be specified as number of bytes. Defaults to `250mb`.
 
 [NOTE]
 ============
-Setting `xpack.reporting.csv.maxSizeBytes` much larger than the default 10 MB limit has the potential to negatively affect the
-performance of {kib} and your {es} cluster. There is no enforced maximum for this setting, but a reasonable maximum value depends
-on multiple factors:
+We recommend using CSV reports to export moderate amounts of data only. The feature enables analysis of data in external tools, but it's not intended for bulk export or to backup {es} data. Setting `xpack.reporting.csv.maxSizeBytes` to larger than 250 MB can negatively affect the performance of your {es} cluster.
```

Comment:
> Setting `xpack.reporting.csv.maxSizeBytes` to larger than 250 MB can negatively affect the performance of your {es} cluster.

If so, I think that sounds like an ES bug. Is there an ES issue which gives some more details of the performance issue here?

Reply:
We have this setting in place as a guardrail/circuit breaker. Kibana report types are generic integrations (CSV is just one of them) that use timeouts as a health check. This circuit breaker helps ensure that a CSV report does not become so large that the report job could time out and be cancelled or retried.

The wording here expresses that CSV export has certain constraints. The guardrails are configurable, but if users must adjust them they must know they are entering a territory that isn't supported. Meaning: if users increase the guardrails and begin to see timeouts or no data being collected, then they should use the ES APIs directly. Without heavy use of guardrails, it's very expensive to have the dev team is responsible for troubleshooting issues like this.

However, I wouldn't say this is a bug in either Elasticsearch or Kibana. Thoughts?

Reply:
Limits are good, sure, but I am not comfortable with saying that we have this one because it might `affect the performance of your {es} cluster`. If there's a Kibana-side limit of 250MiB then that's fine, but please say it's a constraint imposed by Kibana and not ES.

Reply:
Sounds good to me, although I'd phrase it more imperatively:

> Do not increase `xpack.reporting.csv.maxSizeBytes` beyond 250 MB. Instead, create smaller reports, or extract the data you need directly from {es}.

Reply:
We need to convey that the **intent** of the feature is not bulk data export, so Tim's first sentences look good, and I really like Dave's imperative phrasing.  Combine, and I think we're good to go here.

Reply:
Thanks for the feedback! Pushed 30510558e6a